scalding-core/src/main/scala/com/twitter/scalding/ExecutionContext.scala

```diff
@@ -50,7 +50,7 @@ trait ExecutionContext {
     })
   }
 
-  final def buildFlow: Try[Flow[_]] =
+  final def buildFlow: Try[Option[Flow[_]]] =
```

Comment:
Could you document when empty flow returns?

Reply:
sure.

Reply:
done.